### PR TITLE
Meta: add configurable host ports

### PR DESCRIPTION
**Pull request by sungvzer**
This should enable users who already use some of these ports to run the system by setting the appropriate environment variable.
This is related to #16536.

### CODE REVIEW

File: Meta/run.sh [hunk 1]
```diff
@@ -251,11 +251,28 @@ if [ -z "$SERENITY_ETHERNET_DEVICE_TYPE" ]; then
   SERENITY_ETHERNET_DEVICE_TYPE="e1000"
 fi
 
+if [ -z "$SERENITY_HOST_TELNET_PORT" ]; then 
+  SERENITY_HOST_TELNET_PORT=8823
```
Comment (linusg) [inline Meta/run.sh:255]: Four spaces for indentation

**Review by linusg — CHANGES_REQUESTED** [review]

**Comment by timschumi** [comment]
We already allow for selecting a different IP to bind to, do we even need to allow for binding to different ports then?

**Comment by sungvzer** [comment]
> We already allow for selecting a different IP to bind to, do we even need to allow for binding to different ports then?

Sometimes I connect via GDB bound to localhost, and I manually change the SSH port as mentioned in the issue to boot into the system. 

Thing is, the change done to Meta/run.sh sometimes slips into my commits. This should not hinder normal development cycles, just make life easier for those who want to bind services to other ports individually without binding to a different host IP.

File: Meta/run.sh [hunk 2]
```diff
@@ -248,14 +248,31 @@ if [ -z "$SERENITY_DISABLE_GDB_SOCKET" ]; then
...
+
+if [ -z "$SERENITY_HOST_WEBDRIVER_PORT" ]; then 
+    SERENITY_HOST_WEBDRIVER_PORT="8000"
+fi
+
+if [ -z "$SERENITY_HOST_SSH_PORT" ]; then 
+    SERENITY_HOST_WEBDRIVER_PORT="2222"
+fi
+
+# Could not find a service that uses this port by default
```
Comment (konradekk) [inline Meta/run.sh:266]: this is pretty old and probably was used by andreas for development of tcp stack. i guess this is just an artefact to be removed, but could also be treated as a pre-configured port free to be used.

if not deleting, i guess you could name it in some generic way, e.g. `GENERIC`, `DUMMY` or something along those lines…

**Review by konradekk — APPROVED** [review]

File: Meta/run.sh (hunk 2 above)
Comment (sungvzer) [reply Meta/run.sh]: Renamed it to `GENERIC`.

File: Meta/run.sh [hunk 3]
```diff
@@ -248,14 +248,31 @@ if [ -z "$SERENITY_DISABLE_GDB_SOCKET" ]; then
...
+    SERENITY_ETHERNET_DEVICE_TYPE="e1000"
+fi
+
+if [ -z "$SERENITY_HOST_TELNET_PORT" ]; then 
+    SERENITY_HOST_TELNET_PORT="8823"
+fi
+
+if [ -z "$SERENITY_HOST_WEBDRIVER_PORT" ]; then 
+    SERENITY_HOST_WEBDRIVER_PORT="8000"
+fi
```
Comment (linusg) [inline Meta/run.sh:260]: This is the `WebServer` port, primarily.

**Review by linusg — CHANGES_REQUESTED** [review]
Please squash the commits.

**Pull request closed without merging**